README.md

```diff
@@ -113,9 +113,11 @@ To help ensure this plugin is kept updated, new features are added and bugfixes
     - [setUserProperty](#setuserproperty)
   - [Crashlytics](#crashlytics)
     - [setCrashlyticsCollectionEnabled](#setcrashlyticscollectionenabled)
+    - [didCrashOnPreviousExecution](#didCrashOnPreviousExecution)
     - [isCrashlyticsCollectionEnabled](#iscrashlyticscollectionenabled)
     - [setCrashlyticsUserId](#setcrashlyticsuserid)
     - [sendCrash](#sendcrash)
+    - [setCustomKey](#setCustomKey)
```

Comment:
nit : Should it be named something like `setCrashlyticsCustomKey` or `setCrashReportCustomKey` or something like that to make it clearer that this is related to the crash-reporting feature of `firebasex` and not related to , say, FireStore or Remote Config or Analytics ? (otherwise people might mistake it for `setUserProperty` for instance)